shared/common-adapters/meta.js

```diff
@@ -14,8 +14,10 @@ import {
 type Props = {
   title: string,
   style?: StylesCrossPlatform,
+  size?: 'Small',
   color?: string,
   backgroundColor: string,
+  lowercase?: boolean,
```

Comment:
might be better as `noUppercase` so callers can set mixed case stuff freely

Reply:
Good idea!